python/src/ProjectionStrategyImplementation_doc.i.in

```diff
@@ -249,3 +249,16 @@ m : Distribution
 %enddef
 %feature("docstring") OT::ProjectionStrategyImplementation::setMeasure
 OT_ProjectionStrategy_setMeasure_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_ProjectionStrategy_getDesignProxy_doc
+"Accessor to the design proxy.
+
+Parameters
+----------
+designProxy : DesignProxy
```

Comment:
:class:

Reply:
Fixed at 603c19e658a37228afd190e60b6213197baabcef

Reply:
Here is the result:

![image](https://github.com/openturns/openturns/assets/31351465/dda5dfee-5d13-4710-8768-4c731d77b1d5)